README.md

```diff
@@ -67,7 +67,7 @@ To get a local copy up and running follow these simple example steps.
 - To my coding partners, morning session team,reviewers and standup team memebers.
 
```

Comment:
- [ ] In "Acknowledgements" section please include a credit to [Cindy Shin](https://www.behance.net/adagio07) the author of the [original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the Creative Commons License.